targo_test.go

```diff
@@ -32,6 +32,20 @@ var (
 	voidPath          = parentPath + "/void"
 )
 
+//A path without an ending slash will produce a tar
```

Comment:
This is all good but don't you think that it belongs to the README file rather than in the tests?